src/main/kotlin/com/expediagroup/openworld/sdk/core/configuration/OpenWorldClientConfiguration.kt

```diff
@@ -30,7 +30,8 @@ data class OpenWorldClientConfiguration(
     override val key: String? = null,
     override val secret: String? = null,
     override val endpoint: String? = null,
-    val authEndpoint: String? = null
+    val authEndpoint: String? = null,
```

Comment:
Let's keep any added config to the last (switch the order between `authEndpoint` and `requestTimeoutMillis`).

Reply:
Done